### ISSUE TITLE
Some Quenstions about Loss Function

### ISSUE THREAD

**Issue by LCJHust** [PeterTor/sparse_convolution#5]
Hello, I'm reproducing the results in your paper now. I have some questions about the loss function of the network. How can I design the loss function? You have mentioned 'our approach using 5% sparsity during training and testing.', this means the inputs images had 5% sparsity, then the outputs of the network will be more density. What should I compare with the output to get the monitoring signal?
If comparing the output with the input, the predictions will be sparse again?
So I'm so confused. I really hope that you can answer my question. Looking forward to your reply ! Thank you!

**Comment by LCJHust** [comment]
Hello, I'm reproducing the results in your paper now. I have some questions about the loss function of the network. How can I design the loss function? You have mentioned 'our approach using 5% sparsity during training and testing.',  this means the inputs  images had 5% sparsity, then the outputs of the network will be more density.  What should I compare with the output to get the monitoring signal?
If comparing the output with the input, the predictions will be sparse again?
So I'm so confused. I really hope that you can answer my question. Looking forward to your reply ! Thank you!
